Tests/Marketplace/marketplace_services.py

```diff
@@ -1285,6 +1302,10 @@ def prepare_release_notes(self, index_folder_path, build_number, pack_was_modifi
                         return task_status, not_updated_build
                     else:
                         if latest_release_notes in changelog:
+                            #  Check if the modified files are not only release notes
+                            #  If the modified files are only release notes, don't change the timestamp of the entry
+                            if pack_was_modified:
+                                pack_was_modified = not self.only_release_notes_were_modified(modified_files_paths)
```

Comment:
let's move this handling to where that `pack_was_modified` var is first initiated as we want to keep all of this logic in one place.

Reply:
If we do that, in the case of only modifying RNs, the function _enhance_pack_attributes will not update the released date. Is that desired?

Reply:
Yes, if only RN files are changed, the version stays the same code-wise, so no need to update the timing. Can you please check if it has other consequences?